nexus-sdk-js/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "nexus-sdk",
+  "version": "1.0.0",
+  "description": "JavaScript SDK from Nexus",
+  "main": "lib/index.js",
+  "module": "es/index.js",
+  "unpkg": "dist/nexus-sdk.js",
+  "types": "lib/index.d.ts",
+  "license": "MIT",
+  "scripts": {
+    "test": "jest",
+    "lint": "tslint src/**/*",
+    "build": "yarn clean && yarn build:commonjs && yarn build:es && yarn build:umd",
+    "build:commonjs": "tsc --module commonjs --target es5 --outDir lib",
+    "build:es": "tsc --module es2015 --target es5 --outDir es",
+    "build:umd": "rollup -c",
+    "documentation": "typedoc --out documentation/ --exclude \"**/*+(test|spec).ts\" src/",
+    "clean": "rm -fr es/ && rm -fr dist/ && rm -fr lib/"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.1.2",
+    "@babel/core": "^7.1.2",
+    "@babel/preset-env": "^7.1.0",
+    "@types/jest": "^23.3.5",
+    "eslint": "^5.6.0",
+    "jest": "^23.6.0",
+    "rollup": "^0.66.4",
```

Comment:
What's the rationale for using rollup here instead of parcel?

Reply:
It's a library, not a front-end app. We don't have to manage loaders such as CSS, SVG, etc... We need to bundle it into commonjs, umd and es6. Rollup is very simple to setup for that.